lms/templates/static_htmlbook.html

```diff
@@ -93,20 +101,20 @@
     $(document).ready(function() {
         var options = {};
         %if 'url' in textbook:
-            options.url = "${textbook['url']}";
+            options.url = "${textbook['url'] | n, js_escaped_string}";
         %endif
         %if 'chapters' in textbook:
             var chptrs = [];
             %for chap in textbook['chapters']:
-                chptrs.push("${chap['url']}");
+                chptrs.push("${chap['url'] | n, js_escaped_string}");
             %endfor
             options.chapters = chptrs;
         %endif
         %if chapter is not None:
-            options.chapterNum = ${chapter};
+            options.chapterNum = ${chapter | n, dump_js_escaped_json};
```

Comment:
@nedbat: This is causing a unit test failure because `chapter` was a string and it is expecting:
```
options.chapterNum = 2;
```
And it is now getting:
```
options.chapterNum = "2";
```
I can fix this by changing the code to:
```
options.chapterNum = ${int(chapter) | n, dump_js_escaped_json};
```
Question: Can you think of any alternative that would avoid this seemingly needless pain?